### ISSUE TITLE
Size-limited sub-categories for Games: 256b, 4k, 16k, 32k etc.

### ISSUE THREAD

**Issue by tomaes** [demozoo/demozoo#366]
Just like with intros, we could use some categories for all the size-limited games; tagging the size or using the intro prod types feels incoherent to me.

The case for 32k (and 96k):
- 60+ Games on PC and other platforms released at Mekka'96 and the Mekka & Symposium parties up to 2001 alone (and then, they upped it for .kkrieger to 96k; all the Breakpoint parties had those)

The case for 16k:
- 100 or so games from the RGCD compos at csdb alone

The case for 4k:
- 50 or so games from the 4kb games coding competitions at csdb alone

There are quite a few 8k, 1k and 256b games (13 at least) as well. (Here's a good one: http://www.pouet.net/prod.php?which=29286).

One could argue that size and prod types shouldn't be linked in the first place, etc. But let's not go there. :grin:

**Comment by gasman** [comment]
I don't see the problem with setting the production types to 'game' and the appropriate intro type - this way it's consistent with how we'd deal with (for example) 64K musicdisks.

**Comment by tomaes** [comment]
Yes, but intros are not games or music disks. Intros are demos with a size limit. So it feels like a strange workaround for a design flaw to use those for other production types. Pouet had the same problem and somewhat ambiguitively retrofitted the intro types to just say '32k' etc., meaning if nothing else is tagged in union, it must be an intro.

**Comment by asbjornu** [comment]
Is "intro" really a well-defined concept anymore, though? Previously the deal with "intros" was that due to the size-limits, people were unable to make full demos. With modern techniques and hardware, that is no longer the case and some of the most impressive demos of all time fit inside a 4kb or 64kb executable.

So that begs the question: What's important here – the "intro" concept, or the size limit? I'd say the size limit is what really matters and that "intro" is just a (historical) moniker that mostly inaccurately describes the things we label with it.

And if the size limit is what matters, "64k", "4k", "256b" at its own is enough and gives us the flexibility to pair it up with whatever other label we want: "game", "demo", "intro", etc.
